e2e/suite_test.go

```diff
@@ -178,9 +178,10 @@ func (e *E2ETest) provisionCluster() {
 		Pools: []civogo.KubernetesClusterPoolConfig{
 			{
 				Count: 2,
-				Size:  "g4s.kube.xsmall",
+				Size:  "g4s.kube.small",
 			},
 		},
+		FirewallRule: "443;80",
```

Comment:
> [!NOTE]
I have set the firewall_rule based on the following error that occurred during cluster creation.
```
Unable to provision new cluster: Error: Unknown error response - status: 400 Bad Request, code: 400, reason: {"code":"firewall_missing","reason":"Please provide either instance_firewall or firewall_rule"}
Unable to provision new cluster: Error: Unknown error response - status: 400 Bad Request, code: 400, reason: {"code":"firewall_missing","reason":"Please provide either instance_firewall or firewall_rule"}
```